### DISCUSSION TITLE
Impact on battery life

### DISCUSSION THREAD

**Discussion by melyux** [MonitorControl/MonitorControl#1208] [Q&A] [answered]
Since this has to run in the background all the time, how is the impact on battery life? Since external monitors are rarely used when on battery, does anyone know of any mechanism to hook this app to run when charging and quit when on battery?

**Comment by waydabber** [comment, marked as answer]
MC should have negligible battery life effect, it just mostly sits there. There is a process doing basic stuff but the app should never show up as a significant battery drainer in Activity Monitor.
